### DISCUSSION TITLE
0.8 Beta

### DISCUSSION THREAD

**Discussion by TfTHacker** [TfTHacker/obsidian42-text-transporter#14] [Announcements]
0.8 Beta release of "Obsidian42 - Text Transporter"

Hold on to your saddles.. major code refactoring and features. So I expect the need for a quick 8.1 release with bug fixes. But the stuff here is just to darn good to hold back from you crazy testers. 

New
- SUPER ALPHA: Support for tag based file and block search. Just run a command, and select either tag file search or tag block search, and you can then navigate your vault with tags. This is very experimental, not even sure where it will go or what value it offers. but its a starting point. (Thanks dpthomas87)
- Quick Capture supports resolving tags and file names. Not perfect implementation, but it works.  (Thanks @murf)
- Bookmarks can now be configured to add "Copy to" and "Paste to" context menu in editor and all commands list as found in the ribbon button
    - Add a * to the beginning of the line to indicate that a context menu command should be made for this bookmark
    - Example:
      - *directory/myfile.md#TOP
- "Send link of current note" command added. It will send the current file in edit mode as a link into another file. (Thanks  @Kevin Murray)
- "Select line" commanded enhanced to be "Select current line/expand to block" Select line will select current line, and if pressed again will expand to the current block (line and blocks are not always the same thing)

Changes
- Consolidated commands CC and ABI into one. (Thanks FelipeRearden ). Will expand on this command as per your issue.
- Some shortcuts where changed, so some hotkeys may not work and need  to be reassigned

Fixes
-  Copy Block ref for blockquotes now works (Thanks PR-C 🐙)

<hr /><em>This discussion was created from the release <a href='https://github.com/TfTHacker/obsidian42-text-transporter/releases/tag/0.8'>0.8 Beta</a>.</em>

**Comment by Murf** [comment]
Wrong Murf
